pom.xml

```diff
@@ -676,5 +686,9 @@
                 <enabled>true</enabled>
             </snapshots>
         </repository>
+        <repository>
+            <id>cloudera-repo</id>
```

Comment:
I would like to move this down into the shims/spark311cdh/pom.xml file and remove it here, to protect us from accidentally pulling something from cdh repo.